src/messages.cpp

```diff
@@ -236,12 +236,12 @@ MLSPlaintext::MLSPlaintext(bytes group_id_in,
                            ContentType content_type_in,
                            bytes authenticated_data_in,
                            const bytes& content_in)
-  : group_id(std::move(group_id_in))
+  : wire_format(WireFormat::mls_ciphertext)
+  , group_id(std::move(group_id_in))
   , epoch(epoch_in)
```

Comment:
could we add a single line comment that explains the reason this version of the constructor makes wire_format to be mls_ciphertext. I feel it will be handy on quick read. thanks